### ISSUE TITLE
Golden Pantaloons Corrupt Save

### ISSUE THREAD

**Issue by enderandrew** [EE-Mod-Setup/EE-Mod-Setup#89]
I got to Friendly Arm Inn, and saves after that were corrupted and wouldn't load. I've run into this before in the past. This time I was determined to track this down a bit more. I went to a save before Friendly Arm Inn, and saved frequently, reloading and testing the saves. I found the issue where the game breaks for me is if I get Golden Pantaloons. This is something from vanilla. The game doesn't crash or anything. I can keep playing for hours later, but any save after that is corrupt for some reason.
[EE-Debug.zip](https://github.com/EE-Mod-Setup/EE-Mod-Setup/files/12270518/EE-Debug.zip)

**Comment by enderandrew** [comment]
I get this isn't an issue with the tool. Perhaps I should have opened these under the Github repository for the Sandrah Saga and not here. I can't get support for any BGEE issues anywhere else if I use the Sandrah mod and sadly your support forums are no longer online.

**Comment by RoxanneSHS** [comment]
There is nothing that Sandrah mod does with the pantaloons?
However the mod sets a global once you received them so that her parcel later could retrieve it.
Check that C:GetGlobal("ItemGopant","GLOBAL") is set to 1 after you received the pants. That is all Sandrah does with it...no real connection to your issue but my only clue.

(There is some other mod I noticed recently that did something with pantaloons later in the game but I forgot which one it was and what the details were - I deleted that installation, so I do not have details anymore.
